dequeue/dequeue.go

```diff
@@ -44,8 +44,16 @@ func New[T ingest.Identifiable](webhookURL string, c ingest.Client[T], s storage
 	if l == nil {
 		l = log.NewNopLogger()
 	}
+	ic := &client[T]{
+		Client: c,
+		operationsTotal: promauto.With(r).NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "ingest_operations_total",
```

Comment:
Maybe we should clarify what kind of operation this was:
```suggestion
				Name: "ingest_client_operations_total",
```